FruityUI/DeviceImages.plist

```diff
@@ -2,10 +2,272 @@
+	<key>tartarus</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/228/228_tartarus.png</string>
+	<key>deathstalker_expert</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/49/49_razer_deathstalker.png</string>
+	<key>blackwidow_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/279/279_blackwidow_chroma.png</string>
+	<key>deathstalker_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/665/665_deathstalker_chroma.png</string>
+	<key>blade_stealth</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/667/667_blade_stealth_2016_6500u.png</string>
+	<key>orbweaver_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/607/607_orbweaver_chroma.png</string>
+	<key>tartarus_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/598/598_tartarus_chroma.png</string>
+	<key>blackwidow_chroma_te</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/571/571_blackwidow_tournament_edition_chroma.png</string>
+	<key>blade_qhd</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/736/736_blade_pro_2016.png</string>
+	<key>blade_pro_2016</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/736/736_blade_pro_2016.png</string>
+	<key>blackwidow_overwatch</key>
+	<string>https://assets.razerzone.com/eeimages/products/23326/overwatch-razer-gallery-5.png</string>
+	<key>blackwidow_ultimate_2016</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/616/616_blackwidow_ultimate_2016.png</string>
+	<key>blackwidow_x_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/716/716_blackwidow_x_chroma.png</string>
+	<key>blackwidow_x_ultimate</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/718/718_blackwidow_x_ultimate.png</string>
+	<key>blackwidow_x_chroma_te</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/717/717_blackwidow_x_tournament_edition_chroma.png</string>
+	<key>ornata_chroma</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/727/727_ornata_chroma.png</string>
+	<key>ornata</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/726/726_ornata.png</string>
+	<key>blade_stealth_2016</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/667/667_blade_stealth_2016_6500u.png</string>
+	<key>blackwidow_chroma_v2</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/1179/1179_blackwidow_chroma_v2_alt.png</string>
+	<key>blade_2016</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/736/736_blade_pro_2016.png</string>
+	<key>blade_pro_2017</key>
+	<string>https://assets.razerzone.com/eeimages/support/products/1200/1200_blade_pro_2017.png</string>
+	<key>huntsman_elite_hw</key>
+	<string></string>
+	<key>huntsman_hw</key>
```

Comment:
On devices that end with `_sw` or `_hw`, only one key is needed, the one without the `_hw`/`_sw` part, as they both refer to the same device - for instance, if you look at the Huntsman Elite, both `huntsman_elite_hw` and `huntsman_elite_sw` exist, but in this file there is only an entry for `huntsman_elite`. 

Would you mind merging those tags? :)